src/main/groovy/io/openliberty/tools/gradle/tasks/AbstractServerTask.groovy

```diff
@@ -555,9 +551,9 @@ abstract class AbstractServerTask extends AbstractTask {
         serverTask.setInstallDir(installDir)
         serverTask.setUserDir(getUserDir(project, installDir))
 
-        def serverOutputDir = getServerOutputDir(project)
-        if (serverOutputDir != null) {
-            serverTask.setOutputDir(new File(serverOutputDir))
+        def outputDir = getOutputDir(project)
+        if (outputDir != null) {
```

Comment:
How would getOutputDir ever return null now?

Reply:
It shouldn't. I'll fix that.